### ISSUE TITLE
build errors when building the documentation

### ISSUE THREAD

**Issue by kbilsted** [mvno/Okanshi#62]
output from running `.\build.cmd` 

```
Creating C:\src\Okanshi\docs\output\reference
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.StopwatchExtensions.Time because: not a named type
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.AverageGauge because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.BasicCounter because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.BasicGauge because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.BasicTimer because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.DecimalGauge because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.DoubleCounter because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.DoubleGauge because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.HealthCheck because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.LongGauge because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.LongTaskTimer because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.MaxGauge because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.MinGauge because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.PeakCounter because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.PerformanceCounter because: the type '[]`1' does not have a qualified name
FSharp.Formatting Error: 0 : Error while building member-name for Okanshi.OkanshiMonitor.DefaultTags because: the type '[]`1' does not have a qualified name
FSharp.Formatting Information: 0 : Starting razor engine
```

**Comment by kichristensen** [comment]
This is apparently an open issue in FSharp.Formatting, see https://github.com/fsprojects/FSharp.Formatting/issues/364. Waiting for it to been fixed

**Comment by kichristensen** [comment]
This has been fixed while updating the build script in 0d3a3229eb5894eea4e2746f54c9dafdb752496e
